src/main/java/org/llorllale/cactoos/matchers/ScalarHasValue.java

```diff
@@ -81,10 +81,6 @@ protected boolean matchesSafely(final Scalar<T> item) {
     @Override
     protected void describeMismatchSafely(final Scalar<T> item,
         final Description description) {
-        description
```

Comment:
@dgroup what does this change have to do with Assertion?

Reply:
@llorllale because this `appendText(was)` caused an error when `was` word was duplicated twice in `actual` section when hamcrest throws the AsserteionError.